addons/flagger/flagger.yaml

```diff
@@ -8,9 +8,9 @@ metadata:
   annotations:
     sidecar.istio.io/inject: "false"
     appversion.kubeaddons.mesosphere.io/flagger: "0.19.0"
-    catalog.kubeaddons.mesosphere.io/addon-revision: "0.19.0-1"
```

Comment:
If we bump the chartReference.version to `0.20.1`, should we update this and the `appversion.kubeaddons.mesosphere.io/flagger` to `0.20` too?

Reply:
I think it was a coincidence that the app and chart version were the same. Chart version 0.20.1 still deploys flagger 0.19.0: https://github.com/mesosphere/charts/blob/659d0ddada7a74e57e6a73f5a0de01fcf633b151/staging/flagger/Chart.yaml#L3-L4

Reply:
I agree with @branden. `appversion` is the internal app version inside of the chart.